docs/source/releases.rst

```diff
@@ -0,0 +1,79 @@
+Release History
+###############
+
+v1.0.0 (2022-09-28)
+===================
+
+API Changes
+-----------
+- LightpathState combines transmission and output_branch into a single
+  field called output, a mapping from output_branch to transmission on
+  that branch
```

Comment:
I think this point is an API change between two non-tagged versions, I think that means it can be skipped here? E.g. `LightpathState` didn't exist at all in the previous tag

Reply:
True, I'll fix that up